src/boot/efi/boot.c

```diff
@@ -1950,6 +1988,32 @@ static VOID config_entry_add_linux(
         uefi_call_wrapper(linux_dir->Close, 1, linux_dir);
 }
 
+static BOOLEAN config_entry_add_signing_keys(
+                Config *config,
+                CHAR16 *name,
+                CHAR16 *keys_dir) {
+        ConfigEntry *entry;
+        CHAR16 *title = NULL, *id = NULL;
+
+        id = PoolPrint(L"signing-keys-%s", name);
+        title = PoolPrint(L"Enroll signing keys: %s", name);
```

Comment:
hmm, now this makes me wondering if we should display "Signing keys" or "Signature keys" here?

I guess, given that we don't actually *sign* here, but just verify signatures it probably is a bit nicer to say "signature keys" here? (But i am not a native speaker, but i kinda feel that would make more sense)